src/main/java/org/jboss/pnc/kafkastore/mapper/BuildStageRecordMapper.java

```diff
@@ -90,4 +106,18 @@ private String buildId(String processContext) {
             return processContext;
         }
     }
+
+    @GET
+    @Produces(MediaType.TEXT_PLAIN)
```

Comment:
I think it's good to track errors. But this is not an endpoint, it's a mapper. You need to move these endpoints to the MainRest class.